package.json

```diff
@@ -52,7 +52,7 @@
     "express-validator": "^6.4.0",
     "joi": "^17.5.0",
     "knex": "^0.95.14",
-    "loglevel": "^1.6.8",
+    "loglevel": "^1.8.0",
```

Comment:
:) @MundhadHarsh what's the motivation to upgrade this lib

Reply:
@dadiorchen it was not importing log from loglevel that's why I had to upgrade